packages/webview_flutter/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## 0.3.10+4
+
+* Calling destroy on Android webview when flutter webview is getting disposed.
+
 ## 0.3.10+3
 
-* Don't log an unknown setting key error for 'debuggingEnabled' on iOS.
```

Comment:
please revert

Reply:
Sorry about confusion with changelog. I added `*` to that.